graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/nodes/access/CreateNonEnumerableDataPropertyNode.java

```diff
@@ -0,0 +1,48 @@
+package com.oracle.truffle.js.nodes.access;
+
+import com.oracle.truffle.api.dsl.Cached;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.object.DynamicObject;
+import com.oracle.truffle.js.nodes.JavaScriptBaseNode;
+import com.oracle.truffle.js.runtime.Errors;
+import com.oracle.truffle.js.runtime.JSContext;
+import com.oracle.truffle.js.runtime.JSRuntime;
+import com.oracle.truffle.js.runtime.objects.JSAttributes;
+
+public abstract class CreateNonEnumerableDataPropertyNode extends JavaScriptBaseNode {
```

Comment:
Since it's almost identical, I think you can merge this node with `CreateDataPropertyNode` and add a boolean _enumerable_ flag instead.

Reply:
Is fixed in commit e87316c.